dotnet/src/Experimental/Agents/AgentMessage.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Experimental.Agents;
+
+/// <summary>
+/// Represents agent message used by agents inherited from <see cref="KernelAgent" /> class.
+/// </summary>
+public class AgentMessage
```

Comment:
Do we need agent message if we already have ChatMessageContent?

Reply:
Agreed, offline, to drop the AgentMessage type and use ChatMessageContent instead.